src/Symfony/Component/Messenger/Asynchronous/Middleware/SendMessageMiddleware.php

```diff
@@ -34,14 +34,15 @@ public function __construct(SenderLocatorInterface $senderLocator, array $messag
     }
 
     /**
+     * @param Envelope $envelope
+     *
      * {@inheritdoc}
      */
-    public function handle($message, callable $next)
+    public function handle($envelope, callable $next)
     {
-        $envelope = Envelope::wrap($message);
```

Comment:
Could this introduce any BC break?

Reply:
Nop, because `EnvelopeAwareInterface`, the contract is that it gets an `Envelope`.